_posts/2020-02-19-js-the-language-meetup.md

```diff
@@ -0,0 +1,135 @@
+
+<img src="/img/posts/2020-02-19-js-the-language-meetup-async-await/socializing.jpg" alt="Socializing" class="centered" />
+
+After having some food and drinks, we start an introduction round. Everything is very informal and its a good start to make people talk. Normally people say where they came from or share experience they've made with JavaScript so far. That was the point as we've realized how broad our audience actually is. Some people had barely started with programming, some others were actively using JavaScript over 15 years. That turned out not to be a problem at all. In order to bring everyone on the same page about async/await, we've started by reminding ourself the general knowledge about this language construct and what it is normally used for. Afterward we've started to collect all the things that people would be interested to know in context of async/await:
+
+<img src="/img/posts/2020-02-19-js-the-language-meetup-async-await/collect-ideas.jpg" alt="Collect ideas" class="centered" />
+
+Process of collecting ideas and questions went quite actively and proposals were coming from participants of all levels. It seems like it took us less than 10 minutes to fill up the flipchart with lots of exciting topics to tackle:
+
+<img src="/img/posts/2020-02-19-js-the-language-meetup-async-await/ideas-list.jpg" alt="Ideas List" class="centered" />
+
+We've tried to answer, present and learn through a dynamic code session. We've started by checking what can and cannot be "awaited". Should we use just async functions as await expressions? Starting from that, we've tried to await almost everything (sync functions, values, generators, code blocks, exceptions and so on). That was lots of fun and many wow-effects, especially when a code-execution went not as many people would expect it to go.
+
+Then, we focused on the relationship between async functions and promises. We've covered topics such as implicit promises, error handling and chaining of async functions.
+
+As the last test, we've checked the behavior of generators as an await expression. Nobody was certain if it returns a yielded value from the generator or will it return the generator itself.
+
+The test cases for all arisen questions can be found in the code snippet below this blogpost. There you can take a deep look on all the experiments we did.
+
+Finally, it is important to say that the topic for the next meetup is
+already chosen. In order to do so, we've started by collecting all ideas that people would be interested to hear about:
+
+<img src="/img/posts/2020-02-19-js-the-language-meetup-async-await/next-topics.JPG" alt="Next Topics" class="centered" />
+
+This wishlist was filled within a couple of minutes as people started to name all topics that they would like to learn or know more about. Here we've experienced again a high participation rate of all people no matter how long they have been using JavaScript. The final list of all topics looked liked this:
```

Comment:
to name all the topics they would like to...

The final list of the topics looked like...